streaming/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@mastodon/streaming-server",
```

Comment:
Since it's being split out to a org namespace, would it make sense to just move it out of this repo, and then reference the package or Dockerfile in this repo

Reply:
For now, I don't think so. It's a private package not intended to be published. Additionally, how the streaming server works is explicitly coupled to how the rest of Mastodon works, and the contract between them isn't (yet) strictly defined.